src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml.cpp

```diff
@@ -493,6 +494,25 @@ ::Visibility GraphingCalculator::ManageEditVariablesButtonVisibility(unsigned in
 
 void CalculatorApp::GraphingCalculator::ActiveTracing_Checked(Platform::Object ^ sender, Windows::UI::Xaml::RoutedEventArgs ^ e)
 {
+    if (!m_cursorShadowInitialized)
+    {
+        this->FindName(L"TraceCanvas");
+
+        // add shadow to the trace pointer
+        AddTracePointerShadow();
+
+        // hide the shadow in high contrast mode
+        CursorShadow->Visibility = m_accessibilitySettings->HighContrast ? ::Visibility::Collapsed : ::Visibility::Visible;
+
+        m_accessibilitySettings->HighContrastChanged +=
+            ref new TypedEventHandler<AccessibilitySettings ^, Object ^>(this, &GraphingCalculator::OnHighContrastChanged);
+
+        Canvas::SetLeft(TracePointer, TraceCanvas->ActualWidth / 2 + 40);
+        Canvas::SetTop(TracePointer, TraceCanvas->ActualHeight / 2 - 40);
+
+        m_cursorShadowInitialized = true;
```

Comment:
Let's move this one to the top of this block to debounce click events, `FindName` can be "slow" sometimes

Reply:
The control needs to be loaded for the rest of this function to work. Also this is a checked event which shouldn't be firing rapidly.